.github/workflows/cd-{{app_name}}.yml.jinja

```diff
@@ -1,7 +1,7 @@
 name: Deploy {{ app_name }}
 # Need to set a default value for when the workflow is triggered from a git push
 # which bypasses the default configuration for inputs
-run-name: Deploy ${{'{{'}}inputs.version || 'main' {{'}}'}} to {{ app_name }} ${{'{{'}} inputs.environment || 'dev' {{'}}'}}
+run-name: Deploy ${{'{{'}} inputs.version || github.sha {{'}}'}} to {{ app_name }} ${{'{{'}} inputs.environment || 'dev' {{'}}'}}
```

Comment:
Could potentially use `github.ref` here instead for a more skimmable name, but lean towards using what is ultimately the input.